spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ProxyRouteLocator.java

```diff
@@ -238,6 +250,13 @@ public void resetRoutes() {
 		return values;
 	}
 
+	protected String mapRouteToService(String serviceId) {
+		if(this.serviceRouteMapper != null) {
```

Comment:
Ok, nice feedback. 
I wasn't too confident about my first integration in ProxyRouteLocator constructor.
Just to be sure : 
By "the default" do you mean a SimpleServiceRouteMapper class that just return the serviceId?
Can I add ServiceRouteMapper parameter in ProxyRouteLocator constructor?